tests/unit/steps/test_plan.py

```diff
@@ -588,7 +651,7 @@ def test_verify_highest_release_achieved(o7k_release, series):
         (
             OpenStackRelease("ussuri"),
             OpenStackRelease("ussuri"),
-            "Please, upgrade control-plane before data-plane",
+            "Please upgrade control-plane before data-plane",
```

Comment:
I'm curious why this test needed updating - was it passing before with the comma?

Reply:
we missed `assert` in line 672

Reply:
Ah right, thanks :+1: Nice catch for the tests :+1: